File: code-reader-kt/src/main/java/com/loopeer/codereaderkt/api/ServiceFactory.java
package com.loopeer.codereaderkt.api;

import com.loopeer.codereaderkt.api.service.GithubService;
import com.loopeer.codereaderkt.api.service.GithubServicej;

public class ServiceFactory {

    public static GithubService getGithubService() {
        return ServiceUtils.getApiService().getRetrofit().create(GithubService.class);
    }


}

File: code-reader-kt/src/main/java/com/loopeer/codereaderkt/api/ServiceFactory.kt
package com.loopeer.codereaderkt.api

import com.loopeer.codereaderkt.api.service.GithubService


class ServiceFactory {

    @Synchronized fun getGithubService(): GithubService {
        return ServiceUtil().getApiService().retrofit.create(GithubService::class.java)
    }

    //这个注释符号代表java中的static相似
}